src/content-scripts/main.ts

```diff
@@ -11,7 +11,7 @@ interface CheckProdResponse {
 
 chrome.runtime.sendMessage({
   command: CHECK_PRODUCTION,
-  args: { location: window.location },
+  args: { hostname: window.location.hostname },
```

Comment:
Someone may use `:8000` for `production-site` and `:8080` for `staging-site` with the same public IP address. Therefore, please use `location.host` instead of `location.hostname`.

The `hostname` does not include the port number. For example:

```
URL: http://192.168.0.1:8000

location.host => 192.168.0.1:8000
location.hostname => 192.168.0.1
```

Reply:
Got it. Please check the PR again (y)